### DISCUSSION TITLE
[custom_components.mail_and_packages.helpers] Error logging into IMAP Server: b'LOGIN failed.' (username/App password correct)

### DISCUSSION THREAD

**Discussion by ThrashinVictim** [moralmunky/Home-Assistant-Mail-And-Packages#981] [Q&A] [answered]
Getting this when trying to add my outlook account.
[custom_components.mail_and_packages.helpers] Error logging into IMAP Server: b'LOGIN failed.'
I am using an app password and I know both username/password is correct. 
Is there something else I am missing?

**Comment by ThrashinVictim** [comment]
Also adding that this is outlook and settings are correct as well.

**Comment by firstof9** [comment]
What email provider?

**Reply by ThrashinVictim** [reply]
Outlook. I added it in a response above.

**Reply by firstof9** [reply]
Do you mean office365?

**Reply by ThrashinVictim** [reply]
I was using outlook.com

**Reply by ThrashinVictim** [reply]
It just stopped working and now just states 
Unable to connect or login to the mail server. Please check the log for details.

**Reply by firstof9** [reply]
> I was using outlook.com

Yup this is office365.

LOGIN failed means they are not accepting your username/password anymore.

**Reply by ThrashinVictim** [reply]
Ok so since I tried multiple created APP passwords and tried them, how do I move forward?

**Reply by ThrashinVictim** [reply]
Or did MS change something with app passwords?

**Comment by ThrashinVictim** [comment, marked as answer]
Well Found out MS is deprecating APP Passwords. So seems this is an MS issue not a Mail and Packages issue
